src/views/AccountDetail.vue

```diff
@@ -1,104 +1,83 @@
-                    <TableListView :data="namespaceList" />
-                </template>
-            </Card>
-
-            <!-- Transactions -->
-            <Card
-                class="card-f card-full-width"
-                :loading="loading"
-            >
-                <template #title>
-                    {{transactionsTitle}}
-                </template>
-
-                <template #control>
-                    <DropDown
-                        :value="selectedTransactionType"
-                        :options="transactionTypes"
-                        @change="changeTransactionType"
-                    />
-                </template>
-
-                <template #body>
-                    <TableListView :data="filteredTransactionList" />
-                </template>
-            </Card>
-        </div>
+  <div class="page">
+    <div class="page-content-card-f">
+      <!-- Account Detail -->
+      <Card class="card-f card-full-width" :loading="loading" :error="error">
+        <template #title>
+          {{ detailTitle }}
+        </template>
+
+        <template #body>
+          <TableInfoView :data="accountInfo" />
+        </template>
+
+        <template #error> Account {{ address }} does not exist </template>
+      </Card>
+
+      <Card class="card-f card-full-width" :loading="loading">
+        <!-- Activity -->
+        <template #title>
+          {{ activityBuckets }}
+        </template>
+
+        <template #body>
+          <TableListView :data="activityBucketList" />
+        </template>
+      </Card>
+
+      <!-- MultiSig Cosignatories -->
+      <Card class="card-f card-adaptive" :loading="loading">
+        <template #title>
+          {{ cosignatoriesTitle }}
+        </template>
+
+        <template #body>
+          <TableListView :data="accountMultisigCosignatories" />
+        </template>
+      </Card>
+
+      <!-- Mosaics -->
+      <Card class="card-f card-adaptive" :loading="loading">
+        <template #title>
+          {{ mosaicsTitle }}
+        </template>
+
+        <template #body>
+          <TableListView :data="mosaicList" />
+        </template>
+      </Card>
+
+      <!-- Namespaces -->
+      <Card class="card-f card-adaptive" :loading="loading">
+        <template #title>
+          {{ namespacesTitle }}
+        </template>
+
+        <template #body>
+          <TableListView :data="namespaceList" />
+        </template>
+      </Card>
+
+      <!-- Transactions -->
+      <Card class="card-f card-full-width" :loading="loading">
+        <template #title>
+          {{ transactionsTitle }}
+        </template>
+
+        <template #control>
+          <DropDown
+            :value="selectedTransactionType"
+            :options="transactionTypes"
+            @change="changeTransactionType"
+          />
+        </template>
+
+        <template #body>
+          <TableListView :data="filteredTransactionList" />
+        </template>
+      </Card>
```

Comment:
All the other Vue templates use 4-space indentation now, can you undo the formatting changes to this file? See [Card.vue](https://github.com/nemfoundation/nem2-explorer/blob/master/src/components/containers/Card.vue) for a guide.